src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System;
+using System.Diagnostics;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Numerics.Tests
+{
+    public class PerformanceTests
+    {
+        private readonly Random _random;
+        private readonly ITestOutputHelper _output;
+
+        public PerformanceTests(ITestOutputHelper output)
+        {
+            _random = new Random(1138);
+            _output = output;
+        }
+
+        [Theory]
+        [ActiveIssue("PerformanceTest")]
+        [InlineData(1000000, 16, 16)]
+        [InlineData(1000000, 64, 64)]
+        [InlineData(1000000, 256, 256)]
+        [InlineData(1000000, 1024, 1024)]
+        [InlineData(100000, 4096, 4096)]
+        [InlineData(100000, 16384, 16384)]
+        [InlineData(100000, 65536, 65536)]
+        public void Add(int count, int leftBits, int rightBits)
+        {
+            RunBenchmark(count, leftBits, rightBits, (l, r) => BigInteger.Add(l, r));
+        }
+
+        [Theory]
+        [ActiveIssue("PerformanceTest")]
+        [InlineData(1000000, 16, 16)]
+        [InlineData(1000000, 64, 64)]
+        [InlineData(1000000, 256, 256)]
+        [InlineData(1000000, 1024, 1024)]
+        [InlineData(100000, 4096, 4096)]
+        [InlineData(100000, 16384, 16384)]
+        [InlineData(100000, 65536, 65536)]
+        public void Subtract(int count, int leftBits, int rightBits)
+        {
+            RunBenchmark(count, leftBits, rightBits, (l, r) => BigInteger.Subtract(l, r));
+        }
+
+        [Theory]
+        [ActiveIssue("PerformanceTest")]
+        [InlineData(1000000, 16, 16)]
+        [InlineData(1000000, 64, 64)]
+        [InlineData(1000000, 256, 256)]
+        [InlineData(100000, 1024, 1024)]
+        [InlineData(10000, 4096, 4096)]
+        [InlineData(1000, 16384, 16384)]
+        [InlineData(100, 65536, 65536)]
+        public void Multiply(int count, int leftBits, int rightBits)
+        {
+            RunBenchmark(count, leftBits, rightBits, (l, r) => BigInteger.Multiply(l, r));
+        }
+
+        [Theory]
+        [ActiveIssue("PerformanceTest")]
+        [InlineData(1000000, 16)]
+        [InlineData(1000000, 64)]
+        [InlineData(1000000, 256)]
+        [InlineData(100000, 1024)]
+        [InlineData(10000, 4096)]
+        [InlineData(1000, 16384)]
+        [InlineData(100, 65536)]
+        public void Square(int count, int bits)
+        {
+            RunBenchmark(count, bits, v => BigInteger.Multiply(v, v));
```

Comment:
Makes sense then.  Thanks.  Maybe just add a comment here to that effect so that future passerby's understand why this may appear duplication with the above.